src/components/SecondaryPanes/Frames/Group.js

```diff
@@ -14,12 +14,11 @@ import "./Group.css";
 import FrameComponent from "./Frame";
 
 import type { LocalFrame } from "./types";
-import type { Frame } from "../../../types";
 import Badge from "../../shared/Badge";
 
-type FrameLocationProps = { frame: Frame };
+type FrameLocationProps = { frame: LocalFrame };
 function FrameLocation({ frame }: FrameLocationProps) {
-  const library = getLibraryFromUrl(frame);
+  const library = frame.library || getLibraryFromUrl(frame);
```

Comment:
could we get away w/ just `getLibraryFromUrl`

Reply:
We cannot because getting the library is no longer as simple as getting the library from the URL. However, I'm confident we can get away with `frame.library`.